docs/dev/placement-api.md

```diff
@@ -0,0 +1,69 @@
+# Node Placement configuration for Template Validator and Node Labeller
+
+1. The cluster admin can modify the Template Validator/Node Labeller CR to inform what nodes the respective operator pods should be deployed on.
+2. Template Validator pods are placed on nodes that expect infra components according to the HCO API. 
+
+```
+Example TV CR with empty Affinity, Node Selector and Toleration values:
+
+apiVersion: ssp.kubevirt.io/v1
+kind: KubevirtTemplateValidator
+metadata:
+  name: kubevirt-template-validator
+spec:
+  affinity: {}
+  nodeSelector: {}
+  tolerations: []
+```
+
+3. Node Labeller pods are placed on nodes that expect workloads according to the HCO API
+
+```
+Example NL CR with empty Affinity, Node Selector and Toleration values:
+
+apiVersion: ssp.kubevirt.io/v1
+kind: KubevirtNodeLabellerBundle
+metadata:
+  name: kubevirt-node-labeller-bundle
+spec:
+  affinity: {}
+  nodeSelector: {}
+  tolerations: []
+```
+
+4. The HCO CR looks like this
```

Comment:
HCO shouldn't be mentioned here, HCO is a consumer of our API.

Reply:
Removed HCO